.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -26,17 +25,18 @@ assignees: ''
 - [ ] Ensure that outstanding [backport PRs] are merged
 - [ ] Consider building new [cilium-runtime images] and bumping the base image
       versions on this branch
-- [ ] Move any unresolved issues/PRs from old release project into the newly
-      created release project
 - [ ] Push a PR including the changes necessary for the new release:
   - [ ] Pull latest branch
-  - [ ] Run `contrib/release/start-release.sh'
+  - [ ] Run `contrib/release/start-release.sh`
   - [ ] (If applicable) Update the `cilium_version` and `cilium_tag` in
         `examples/getting-started/Vagrantfile`
   - [ ] Run `Documentation/check-crd-compat-table.sh vX.Y` and if needed, follow the
         instructions.
   - [ ] Commit all changes with title `Prepare for release vX.Y.Z`
   - [ ] Submit PR (`contrib/release/submit-release.sh`)
+- [ ] Execute `release --current-version X.Y.Z --next-dev-version X.Y.W` to automatically
+      move any unresolved issues/PRs from old release project into the new
+      project. (`W` should be calculation of `Z+1`)
```

Comment:
Would be nice to make this part of the start-release step so it's one less moving piece to think about.

Reply:
Not sure if we have the number `z+1` in the current scripts though, that can be non-trivial to calculate in shell scripts :<